### DISCUSSION TITLE
District Scenarios evaluation

### DISCUSSION THREAD

**Discussion by amb1** [architecture-building-systems/CityEnergyAnalyst#3332] [Q&A] [answered]
Hey CEA-Community,

for a district with 36 buildings, I want to evaluate different scenarios:

- As is: decentralized heat supply at every building, fossil fuels, radiator heating (70/50)
- Baseline: decentralized heat supply with brine/water HP's, radiator heating (70/50)
- District-Baseline: centralized brine/water HP with DH, radiator heating (70/50)
- District-Low Temp: centralized brine/water HP with DH, radiator heating (50/35)

The buildings are the same for every scenario, so the demand stays the same.
I want to evaluate the GHG-emissions for the scenarios and see how the different temperature levels of the heat dissipation will have an effect on them.
So after getting to know the program, filling in the (in my opinion) needed parameters, running the different scripts from the dashboard and looking at the results, there are some questions that I am left with:

In the CH-database ASSEMBLIES-HEATING-district heating - heatpump - soil/water the efficency is set to 0.6, which is the same value as the district heating - natural gas-fired boiler. I assumed this as an typo and changed it to 3.7 which is slightly worse than a decentralized HP.
From the results, it seems that the only decisive value that changes something is the efficency for the heatpump and changing the temperature levels on the heat dissipation systems are not relevant.
By looking at the supply temperature of the DH-system, it seems that the temperature is not dropping below 60°C, is there any setting for this or am I missing something else?

I hope that someone has more insights on how the different systems work together and could help me evaluate the systems more precise

**Comment by ShiZhongming** [comment, marked as answer]
Hi @amb1 

The Supply-temperature is not dropping below 60C as it is hard-coded here:
cea > demand>constants.py>line59
TWW_SETPOINT = 60  # dhw tank set point temperature in C

We do not have the option to change it using the dashboard. 
If you have to change it, you can go to the source code.

I hope this helps.

**Reply by amb1** [reply]
thanks for the clarification regarding the hardcoded setpoint.
so what is the use of the air-conditioning-systems/type_hs & type_dhw?
are those temperatures set there not taken into account?

**Comment by ShiZhongming** [comment]
@amb1 regarding the "typo", I have created an issue and we are investigating. #3338
